ui/apps/platform/src/Containers/ComplianceEnhanced/Dashboard/MockData/complianceScanResultsOverview.json

```diff
@@ -0,0 +1,37 @@
+{
+    "scanOverviews": [
+        {
+            "scanStats": {
+                "scanName": "HIPPA Scan",
+                "numberOfChecks": 54,
+                "numberOfFailingChecks": 34,
+                "numberOfPassingChecks": 20,
+                "lastScan": "2023-08-03T04:49:58Z"
+            },
+            "profileName": ["HIPPA"],
+            "clusterId": ["records-database"]
+        },
+        {
+            "scanStats": {
+                "scanName": "CIS Scans",
+                "numberOfChecks": 78,
+                "numberOfFailingChecks": 23,
+                "numberOfPassingChecks": 55,
+                "lastScan": "2023-08-03T04:49:58Z"
+            },
+            "profileName": ["profile-1", "profile-2"],
+            "clusterId": ["cluster-1", "cluster-2", "cluster-3"]
+        },
+        {
+            "scanStats": {
+                "scanName": "PCI Scans",
+                "numberOfChecks": 45,
+                "numberOfFailingChecks": 16,
+                "numberOfPassingChecks": 29,
+                "lastScan": "2023-08-03T04:49:58Z"
+            },
+            "profileName": ["PCI"],
+            "clusterId": ["cluster-1", "cluster-2"]
```

Comment:
Are these property names hypothetical?

For actual responses, what do you think are pro and con of plural if value is array and multiple are possible?

Reply:
Yeah these are hypothetical and I just threw them in there since they won't actually be displayed anywhere on the table. I'm not sure I understand your second question though. Are you referring to the way the table is set up to only display the cluster/profile name if one and only one is included in the array versus showing "2 clusters" like in the example here?

Reply:
Sorry, I meant if responses are similar to these objects, between singular and plural for these keys.

Reply:
Oh gotcha. I just pulled directly from the proto, but I would think plural makes more sense for both of these keys.

Reply:
Then that is a good discussion question for backend team.